testing/openresty/APKBUILD

```diff
@@ -0,0 +1,78 @@
+# Contributor: Yichun Zhang <agentzh@gmail.com>
+# Maintainer: Yichun Zhang <agentzh@gmail.com>
+pkgname=openresty
+pkgver=1.13.6.1
+resty_openssl_ver=1.0.2k
+resty_pcre_ver=8.41
+pkgrel=0
+pkgdesc="Scalable Web Platform by Extending NGINX with Lua"
+url="https://openresty.org/"
+arch="all"
+license="BSD"
+depends="gd geoip libxslt zlib perl !nginx"
+makedepends="gd-dev geoip-dev libxslt-dev linux-headers perl-dev readline-dev zlib-dev libgcc"
+install=""
+subpackages=""
+
+source="https://openresty.org/download/$pkgname-$pkgver.tar.gz
+openssl-$resty_openssl_ver.tar.gz::https://www.openssl.org/source/openssl-$resty_openssl_ver.tar.gz
+pcre-$resty_pcre_ver.tar.gz::https://ftp.pcre.org/pub/pcre/pcre-$resty_pcre_ver.tar.gz"
```

Comment:
This is not acceptable, please use system-provided libressl (or openssl if needed) and pcre.